src/green/Green.jl

```diff
@@ -0,0 +1,42 @@
+"""
+
+Green's function
+
+
+"""
+module Greenfunc
+
+export GreenUR
+using StaticArrays, GreenFunc
+
+struct GreenUR{T<:AbstractFloat}
+    timetype::Symbol
+    spacetype::Symbol
+
+    color::Int
+    timegrid::AbstractArray{T,1}
```

Comment:
time and space grid shoud be more general, the type can be passed in as GreenUR{T, TGT, SGT}